frontend/src/HomePage/TemplateLibraryModal/TemplateDisplay.jsx

```diff
@@ -1,8 +1,16 @@
 import React from 'react';
 import { Container, Row, Badge } from 'react-bootstrap';
+import { allSvgs } from '@tooljet/plugins/client';
 
 export default function TemplateDisplay(props) {
   const { id, name, description, sources } = props?.app ?? {};
+
+  const getSvgIcon = (key, height = 50, width = 50) => {
```

Comment:
Looks like there are other instances of the same function within the app. Can we move it to utils to avoid duplication?